benchmarks/buiter_2016_wedge/buiter_2016_wedge.prm

```diff
@@ -0,0 +1,172 @@
+# This is the input file for the setup for the benchmark described in Buiter et al, 2016
+# Experiment 1 which simulates a stable wedge undergoing translation.
+
+
+#####  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 14400
+set Use years in output instead of seconds = false
+set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Nonlinear solver tolerance             = 1e-9
+set Max nonlinear iterations               = 25
+set CFL number                             = 0.5
+set Maximum time step                      = 1e6
+set Output directory                       = output
+set Timing output frequency                = 1
+set Pressure normalization                 = no
+
+#### Parameters describing the model
+
+# Model geometry
+subsection Geometry model
+  set Model name = box
+  subsection Box
+    set X repetitions = 500
+    set Y repetitions = 100
+    set X extent      = 0.15
+    set Y extent      = 0.035
+  end
+end
+
+# Mesh refinement specifications
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 0
+  set Time steps between mesh refinement = 0
+end
+
+# Boundary classifications
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators   = bottom, top
+end
+
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = left
+  set Prescribed velocity boundary indicators = right:function
+  set Zero velocity boundary indicators = bottom
+  subsection Function
+    set Variable names      = x,y
+    set Function constants  = cm=0.01, h=3600., th=0.002
+    set Function expression = if (y>th, -2.5*cm/h, -(y/th)*2.5*cm/h) ; 0.
+  end
+end
+
+subsection Boundary traction model
+  set Prescribed traction boundary indicators = top: zero traction 
+end
+
+# Temperature boundary conditions
+subsection Boundary temperature model
+  set List of model names = box
+
+  subsection Box
+    set Bottom temperature = 100
+    set Left temperature   = 100
+    set Right temperature  = 100
+    set Top temperature    = 100
+
+  end
+end
+
+# Initial temperature field
+subsection Initial temperature model
+  set Model name = function
+  subsection Function
+    set Function expression = 100
+  end
+end
+
+# Number and name of compositional fields
+subsection Compositional fields
+  set Number of fields = 5
+  set Names of fields  = sand,bound,block,sticky_air,total_strain
+end
+
+# Spatial domain of different compositional fields
```

Comment:
Either here or at the top of the model please add a brief explanation of the values defining the compositional field geometry below.